src/Pages/Sign/InputWithLabel.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from "react"
+import "./InputWithLabel.scss"
+
+class InputWithLabel extends Component {
+  constructor(props){
+    super(props)
+  this.state={}
+}
+  render() {
+    return (
+    <div className="idBox">
```

Comment:
```suggestion
    <div className="InputWithLabel">
```
컴포넌트 최상위 태그의 클래스명은 컴포넌트 이름과 일치시켜주세요